### DISCUSSION TITLE
Number stored as text when formatting excel export with currency and percentages

### DISCUSSION THREAD

**Discussion by licerog** [mini-software/MiniExcel#688] [Q&A]
![image](https://github.com/user-attachments/assets/115e5d14-a878-4600-bee1-722b39af7348)

The above code snippet formats my cells to currency and percentages, but when I open my excel I'm presented with a problem as I receive "Number stored as text" issues with these columns which is affecting my ability to utilize the excel for reporting. Please advice.

**Comment by michelebastione** [comment]
We apologize for the inconvenience and for the late reply. The problem has now been solved. Please let us know if you have further questions related to the issue.
